Week-5/generator.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * 
+ * Exercise 5.1:
+ * Using Async/Await and Generators, create separate functions and achieve the samefunctionality.
+ * Execute 3 callback functions asynchronously, for example doTask1(), doTask2() and doTask3()
+ */
+
+const doTask1 = (ms) => {
+    return new Promise((resolve) => setTimeout(resolve("Task 1 Done"), ms));
+};
+  
+const doTask2 = (ms) => {
+    return new Promise((resolve) => setTimeout(resolve("Task 2 Done"), ms));
+};
+  
+const doTask3 = (ms) => {
+    return new Promise((resolve) => setTimeout(resolve("Task 3 Done"), ms));
+};
+
+const usingAsync = async () => {
+    let task1 = await doTask1(3000);
+    console.log(task1);
+    let task2 = await doTask2(1000);
+    console.log(task2);
+    let task3 = await doTask3(2000);
+    console.log(task3);
```

Comment:
According to the problem statement these, promises should run in parallel.
Please check if we can use something like Promise.all in this case

Reply:
Ohh, looks like I didn't understood problem. Here is updated solution [94b61c9](https://github.com/pesto-students/p8-manish-AkashDPA/pull/6/commits/94b61c9e60d953bc2bc255edab69947b34629916)